csrc/buffer.h

```diff
@@ -598,6 +598,12 @@ class JByteArrayCritical {
     ~JByteArrayCritical();
     unsigned char* get();
 
+#ifdef HAVE_CPP11
+    // deleting copy constructor and copy assignment to satisfy rule of three
```

Comment:
Should we delete the move constructor and move assignment functions as well to comply with the rule of five?

Reply:
You're right. Technically, removing just the copy operations is sufficient. But it's much cleaner to remove all of them. I'm fixing that:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-copy-move-or-destructor-function-define-or-delete-them-all